### ISSUE TITLE
Cannot prepopulate with users

### ISSUE THREAD

**Issue by lennisthemenace** [FreeTAKTeam/FreeTakServer#115]
The current feature of pushing users certs DP to them automatically is really nice!, can the ability to wait for the user to call in be added?

Current Flow:
user created > dp sent to device > if devices isn't connected, don't send
 
Proposed:
user created  > wait until that client is connected then send/or when that device first registers then send

Also a "Send" button would be useful for an admin I believe to re-push the DP to a user.

**Comment by naman108** [comment]
great point's we'll probably implement this in 1.7 along with cert revokation when you delete a user

**Comment by naman108** [comment]
evidently we didn't implement it in 1.7 but I havent forgotten

**Comment by brothercorvo** [comment]
will be implemented in 1.9

**Comment by brothercorvo** [comment]
this could be integrated in certs enrollment

**Comment by naman108** [comment]
yes likely as simple as a new endpoint on the API and a business rule
